com.unity.render-pipelines.universal/Editor/Deprecated.cs

```diff
@@ -6,7 +6,7 @@ namespace UnityEditor.Rendering.Universal
     /// Editor script for a <c>ForwardRendererData</c> class.
     /// </summary>
     [Obsolete("ForwardRendererDataEditor has been deprecated. Use UniversalRendererDataEditor instead (UnityUpgradable) -> UniversalRendererDataEditor", true)]
-    internal class ForwardRendererDataEditor : ScriptableRendererDataEditor
```

Comment:
When was this obsolete?

Reply:
Landed here https://github.com/Unity-Technologies/Graphics/pull/1933
As far as I can tell. 12 months ago